packages/web-runtime/src/components/FeedbackLink.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <oc-button
+    v-oc-tooltip="label"
+    type="a"
+    :href="href"
+    target="_blank"
+    appearance="raw"
+    :aria-label="label"
+  >
+    <oc-icon name="feedback" />
+  </oc-button>
+</template>
+
+<script>
+export default {
+  name: 'FeedbackLink',
+  computed: {
+    href() {
+      return 'https://owncloud.com/web-design-feedback'
```

Comment:
Not sure if out of scope but we could load it from the theme so users can customize and re-use it?

Reply:
If someone wants to have this as a feature we could do so. For now we are the ones who want that user feedback. ;-) So I'd start with the hardcoded link and see how it evolves.